unlock-app/src/middlewares/walletMiddleware.js

```diff
@@ -77,7 +73,7 @@ const walletMiddleware = config => {
       (transactionHash, from, to, input, type, status) => {
         // At this point we know that a wallet was found, because a new transaction
         // cannot be created without it
-        dispatch(gotWallet())
+        dispatch(dismissWalletCheck())
```

Comment:
The original implementation made a distinction between manually dismissed wallet check and automatically dismissed wallet check. The modal doesn't need to know about that, so it just has launch and dismiss actions.